3-platform/order/spec/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.6core.platform</groupId>
+        <artifactId>order</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>order-spec</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.6core.lib</groupId>
+            <artifactId>java-domain-model</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webflux-ui</artifactId>
+            <version>2.3.0</version>
```

Comment:
Maybe it's better to use BOM (dependencyManagement) for this and future dependencies to avoid version incompatibilities?

Reply:
@kateryna-mykh  Hello! we changed it according to Roman remarks. Look at the dev branch.

Reply:
Ok